src/main/java/hudson/plugins/sloccount/SloccountProjectAction.java

```diff
@@ -1,36 +1,33 @@
 package hudson.plugins.sloccount;
 
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
+import hudson.model.Run;
+import hudson.model.Job;
 import hudson.model.Action;
 import hudson.util.ChartUtil;
 import java.io.IOException;
-import java.io.Serializable;
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.StaplerResponse;
 
 /**
  *
  * @author lordofthepigs
  */
-public class SloccountProjectAction implements Action, Serializable {
-    /** Serial version UID. */
-    private static final long serialVersionUID = 0L;
+public class SloccountProjectAction implements Action {
 
```

Comment:
This was exactly what I meant. It looks that ProjectActions are not saved.
Only BuildActions are.